app/Http/Controllers/DashboardEventController.php

```diff
@@ -62,6 +62,28 @@ protected function redirectToEventIfNoRegistrationIsPossible(Event $event)
         }
     }
 
+    /**
+     * Redirect to the event page for the given event if the unregistration is not possible
+     *
+     * @param Event $event
+     *
+     * @return \Illuminate\Http\RedirectResponse|void
+     */
+    protected function redirectToEventIfNoUnregistrationIsPossible(Event $event)
+    {
+        // check if unregistration is already open
+        if ($event->registration_from && $event->registration_from->isFuture()) {
+            Session::flash('error', 'Abmeldung ist noch nicht möglich');
+            return $this->redirectToEvent($event);
+        }
+
+        // check if unregistration is already closed
```

Comment:
We check if the registration is already closed, not if the unregistration is already closed